src/Raven.Server/RavenServer.cs

```diff
@@ -295,7 +295,18 @@ void ConfigureKestrel(KestrelServerOptions options)
                         // default to disabled
                         Configuration.Server.CpuCreditsExhaustionFailoverThreshold ?? -1;
 
-                    var _ = StartMonitoringCpuCredits();
+                    PoolOfThreads.GlobalRavenThreadPool.LongRunning(_ =>
```

Comment:
Need to wait for it to finish on server dispose:
https://github.com/ravendb/ravendb/blob/0568c39f984877569deead635c74a9b35928e094/src/Raven.Server/ServerWide/ServerStore.cs#L1793